src/pages/access_review/VoteSummary.js

```diff
@@ -113,7 +113,7 @@ export const VoteSummary = hh(
 
     memberVote = (vote) => {
       const voteString = fp.isNil(vote.vote.vote) ? 'Pending' : vote.vote.vote ? 'Yes' : 'No';
-      const createDateString = fp.isNil(vote.vote.createDate) ? '' : moment(vote.vote.createDate).format('MM/DD/YY');
+      const updateDateString = fp.isNil(vote.vote.updateDate) ? '' : moment(vote.vote.updateDate).format('MM/DD/YY');
```

Comment:
From what I've seen some votes have ```createDate``` while others have ```updateDate``` (might be an issue of older votes/legacy behavior vs newer votes, not entirely sure). Just to be safe check for both, with ```createDate``` as a fallback if ```updateDate``` is not present.

<img width="816" alt="Screen Shot 2021-04-23 at 10 44 04 AM" src="https://user-images.githubusercontent.com/6664778/115888855-63ca2400-a421-11eb-8ca0-a96d23b710ed.png">